doc/source/how-to/manage-protobuf-definitions.rst

```diff
@@ -0,0 +1,163 @@
+    requires = ["setuptools >= 42.0.0", "wheel", "ansys_tools_protoc_helper"]
+
+Basic python project build step that can be executed locally or within a
+build pipeline:
+
+.. code-block:: python
+
+    pip install -U pip
+    pip install build
+    python -m build
+
+Publishing Python API package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PyPI is the common package manager where API packages are released.
+
+Example nightly build pipeline publishing the python stub package:
+
+.. code-block:: yml
+
+    name: Nightly dev release
+  
+    on:
+      schedule: # UTC at 0300
+        - cron: "0 5 * * *"
+      workflow_dispatch:
+          
+    env:
+      MAIN_PYTHON_VERSION: '3.10'
+    
+    jobs:
+    nightly-release:
+        name: Nightly release package
+        if: github.ref == 'refs/heads/main'
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v3
+    
+        - name: Append date to version tags
+            run: |
+            PACKAGE_DATE=$(date +'%Y%m%d%H%M')
+            sed -i "1s/.$/$PACKAGE_DATE/" ansys/api/<api-name>/VERSION
+        - name: Set up Python
+            uses: actions/setup-python@v4
+            with:
+            python-version: ${{ env.MAIN_PYTHON_VERSION }}
+    
+        - name: Install build requirements
+            run: |
+            pip install -U pip
+            pip install build
+        - name: Build
+            run: python -m build
+    
+        - name: Install
+            run: pip install dist/*.whl
+    
+        - name: Test import
+            run: |
+            mkdir tmp
+            cd tmp
+            python -c "import ansys.api.<api-name>.v0; print('Successfully imported ansys.api.<api-name>.v0')"
+            python -c "from ansys.api.<api-name> import __version__; print(__version__)"
+        - name: Upload to PyPI
+            run: |
+            pip install twine
+            twine upload --skip-existing ./**/*.whl
+            twine upload --skip-existing ./**/*.tar.gz
+            env:
+            TWINE_USERNAME: PAT
+            TWINE_PASSWORD: ${{ secrets.PYANSYS_PyPI_PAT }} 
+            TWINE_REPOSITORY_URL: https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/PyPI/upload
+    
+        - name: Upload packages
+            uses: actions/upload-artifact@v3
+            with:
+            name: ansys-api-<api-name>-packages
+            path: dist/
+            retention-days: 7
+
+PyPI packages follow semantic versioning while gRPC Protobuf API versions typically follow a simplified v*
+versioning pattern. It is not expected to synchronize PyPI package version with Protobuf API version.
+There is no methodology to correlate PyPI package version with exposed gRPC API versions included within
+the package.
+
+
+Consuming the API package within Python
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once the API package has been published to PyPI, a reference can be included within
+the client library build dependencies.
+
+Example poetry configuration:
```

Comment:
Here--because there are two examples, I'd recommend removing the colon and formatting them in bold like this:

**Example ``poetry`` configuration**